### ISSUE TITLE
Can this project be used with kotlin-frontend-plugin?

### ISSUE THREAD

**Issue by markjfisher** [willowtreeapps/assertk#204]
This may be my lack of knowledge on kotlin-frontend-plugin, but when I use assertk with that plugin, during the build I get the following error, which I don't know how to resolve:

```
ERROR in ./node_modules_imported/assertk/assertk.js
Module not found: Error: Can't resolve 'opentest4k' in '/path/to/project/build/node_modules_imported/assertk'
 @ ./node_modules_imported/assertk/assertk.js 3:4-56
```

I tried adding the npm module directly with:

```
kotlinFrontend {
    npm {
        devDependency "opentest4k"
    }
```
but that errors saying it can't find the npm module:

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/opentest4k - Not found
npm ERR! 404
npm ERR! 404  'opentest4k@*' is not in the npm registry.
```

Any advice on using assertk with kotlin-frontend-plugin?

**Comment by evant** [comment]
Good question, I'm no too familiar with the kotlin-frontend-plugin myself. When I get time I'll try to creat a sample project and see what's up. Feel free to link to your own sample that's having issues if you want me to take a look.

**Comment by markjfisher** [comment]
I'll create a minimal project and upload it for testing. cheers

**Comment by markjfisher** [comment]
https://github.com/markjfisher/assertk-frontend

**Comment by evant** [comment]
Interesting, so it seems like it's an issue with runtime-scoped dependencies in the [pom](https://search.maven.org/artifact/com.willowtreeapps.assertk/assertk-js/0.16/jar) file. Changing to api seems to fix it. Can you test with the snapshot dep `0.17-SNAPSHOT`? You'll need to add the sonatype snapshot repo with:
```groovy
repositories {
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
}
```

**Comment by markjfisher** [comment]
That's got it, build works, tests run. Thanks for the update.

**Comment by evant** [comment]
Released in `0.17`
